pkg/identityserver/mock/device_registry.go

```diff
@@ -25,28 +25,31 @@ import (
 type mockISEndDeviceRegistry struct {
 	ttnpb.EndDeviceRegistryServer
 
-	endDevicesMu sync.RWMutex
-	endDevices   map[string]*ttnpb.EndDevice
+	endDevices map[string]*ttnpb.EndDevice
+	sync.Mutex
```

Comment:
Any reason for not using `sync.Map`?

Reply:
Just didn't come to mind, I ended up making the change only on the device registry.